packages/components/src/radio-control/index.js

```diff
@@ -20,6 +20,7 @@ export default function RadioControl( {
 	selected,
 	help,
 	onChange,
+	hideLabelFromVision,
```

Comment:
We should also add this to the `README.md` file like for example here: https://developer.wordpress.org/block-editor/reference-guides/components/input-control/#hidelabelfromvision

Reply:
Thanks for pointing it out! Done in 6de4c8905837461a28a3f785636d46f9ed693d66